lib/services.rb

```diff
@@ -33,6 +33,10 @@ def self.rummager
       disable_cache: true
     )
   end
+
+  def self.content_tagger
```

Comment:
As this is not a service/API, this might be a little confusing that it returns a URL.